src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/CustomItemEffects.java

```diff
@@ -1067,6 +1184,114 @@ public void renderBlockOverlay(DrawBlockHighlightEvent event) {
 							}
 						}
 
+						GlStateManager.depthMask(true);
+						GlStateManager.enableTexture2D();
+						GlStateManager.disableBlend();
+					}
+				} else if ((heldInternal.equals("HOE_OF_GREATEST_TILLING") &&
+					NotEnoughUpdates.INSTANCE.config.itemOverlays.enableHoeOverlay && onPrivateIsland) &&
+					event.target.typeOfHit == MovingObjectPosition.MovingObjectType.BLOCK) {
+					BlockPos target = event.target.getBlockPos();
+					IBlockState targetState = Minecraft.getMinecraft().theWorld.getBlockState(target);
+
+					if (targetState.getBlock() == Blocks.dirt || targetState.getBlock() == Blocks.grass) {
+						GlStateManager.enableDepth();
+						GlStateManager.enableBlend();
+						GlStateManager.tryBlendFuncSeparate(770, 771, 1, 0);
+						GlStateManager.disableTexture2D();
+						GlStateManager.depthMask(true);
+
+						LinkedList<BlockPos> candidates = new LinkedList<>();
```

Comment:
please use an interface type instead of a concrete type when the implementation does not matter

Reply:
also candidatesOld and candidatesNew, you really just gave up with naming these, right?

Reply:
a standard List doesnt have pop